django/db/backends/postgresql/introspection.py

```diff
@@ -207,10 +209,11 @@ def get_constraints(self, cursor, table_name):
                 LEFT JOIN pg_class c2 ON idx.indexrelid = c2.oid
                 LEFT JOIN pg_am am ON c2.relam = am.oid
                 LEFT JOIN pg_attribute attr ON attr.attrelid = c.oid AND attr.attnum = idx.key
+                JOIN pg_namespace nsp ON nsp.oid = c.relnamespace AND nsp.nspname = %s
```

Comment:
You could just do this:

```suggestion
                JOIN pg_namespace nsp ON nsp.oid = c.relnamespace AND nsp.nspname = current_schema()
```